assets/settings/default.json

```diff
@@ -596,6 +596,8 @@
     // when a corresponding project entry becomes active.
     // Gitignored entries are never auto revealed.
     "auto_reveal_entries": true,
+    // Whether the project panel should open automatically when opening a folder.
```

Comment:
> when opening a folder

That's not right, one can open a file too.
That's about startup, it seems? 
Need to update all docs around.